product-design-roles.md

```diff
@@ -205,72 +205,72 @@ Exemplifies the Design Leadership Principles, and serves as an example to everyo
 ### Visual Design
 
 *Associate Product Designer*  
-Demonstrates taste in layout, typography and visual hierarchy, as it pertains to BuzzFeed’s brand and products.
+Demonstrates taste in layout, typography and visual hierarchy. Uses our Design Principles, as well as existing style guides and previous Product Design work to represent the BuzzFeed brand.
 
 *Product Designer*  
-Demonstrates **understanding** of layout, typography and visual hierarchy, as it pertains to BuzzFeed’s brand and products.
+Demonstrates **understanding** of layout, typography and visual hierarchy. Uses our Design Principles, as well as existing style guides and previous Product Design work to represent the BuzzFeed brand **with a holistic consideration for their product's future.**
 
 *Senior Product Designer*  
-Demonstrates and **articulates understanding** of layout, typography and visual hierarchy, as it pertains to BuzzFeed’s brand and products.
+Demonstrates and **articulates understanding** of layout, typography and visual hierarchy. Exhibits strong art direction skills, using existing visual frameworks creatively to produce styles and patterns that are fresh, yet feel visually cohesive with the rest of BuzzFeed's products. **Maintains and suggests evolutions of** our Design Principles and existing style guides with a holistic consideration for their **platform's** future.
```

Comment:
Love "exhibits strong art direction skills", do we define art direction as art directing other designers? (I think it should)

"using existing visual frameworks creatively"  is unclear to me. Is this referring to Solid and our design principles or something else? More traditional graphic design techniques perhaps?

"produce styles and patterns that are fresh, yet feel visually cohesive" do we mean "fresh" in the same sense as the design principles? Curious if we need to call out explicitly if we're already making reference to the Design Principles here.